docs/src/main/sphinx/connector/accumulo.rst

```diff
@@ -24,6 +24,15 @@ JAR file to Accumulo's ``lib/ext`` directory on each TabletServer node.
 
     # TabletServer should pick up new JAR files in ext directory, but may require restart
 
+Requirements
+------------
+
+To connect to Accumulo, you need:
+
+* Accumulo version 2.0.1 or higher.
```

Comment:
The accumulo dependency version is 1.7.4 .. lets use that for now

Reply:
Yes, Accumulo 2.x actually isn't compatible with this connector. I believe it will need split into a separate module/connector with separate client libraries to support 2.x.